rasa/plugin.py

```diff
@@ -73,3 +73,8 @@ def create_tracker_store(  # type: ignore[empty-body]
     event_broker: Optional["EventBroker"],
 ) -> "TrackerStore":
     """Hook specification for wrapping with AuthRetryTrackerStore."""
+
+
+@hookspec  # type: ignore[misc]
```

Comment:
You'll need to parameterize the hookspec, otherwise the result comes back as a tuple of the hook function result, so instead of string you'll get `(string_var, )`:
```suggestion
@hookspec(firstresult=True)  # type: ignore[misc]
```

Reply:
Also note that license_hash will be `None` for pure oss users (those that don't use `rasa-plus` instead) because the hook call will return None rather than hookimpl call.

Reply:
Yea It should probably be two versions of default context one with `license_hash` another without it depending if it's `None`